### PR TITLE
DDF-5371 Adds beta-admin-ui

### PR DESCRIPTION
**Pull request by andrewkfiedler**
#### What does this PR do?
 - Fixes a few typescript issues we had.
 - Adds a beta-admin-ui (dependency on @connexta/kanri).
 - Leaves the old admin ui around, so that people can continue to use that one if they choose.
 - Adds a link in the old admin ui so people can find the beta admin ui.
 - Note that @connexta/kanri allows extensions if necessary.

#### How should this be tested?
Verify the old admin ui works as expected.  Verify catalog ui works as expected.

Follow the link to the new beta admin ui in the old admin ui and give it a whirl.

#### Any background context you want to provide?
More context at https://github.com/connexta/kanri and see commit message.  Both places have most of the details.

Main motivation is removing our dependencies on Backbone / Marionette / etc., and making it easier to maintain and build upon.

#### What are the relevant tickets?
Fixes: #5371 

#### Screenshots
Screen Shot 2019-09-24 at 1 48 44 PM
Screen Shot 2019-09-24 at 1 51 33 PM

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: ui/packages/beta-admin-ui/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
```
Comment (Bdthomson) [inline ui/packages/beta-admin-ui/tsconfig.json]: Have we talked any more about auto generating this file in ace?

File: ui/packages/beta-admin-ui/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  "compilerOptions": {
+    "jsx": "react",
+    "allowJs": true,
```
Comment (Bdthomson) [inline ui/packages/beta-admin-ui/tsconfig.json]: Do you really want this one true?

**Comment by Bdthomson** [comment]
I think you're missing some changes to the `ui/yarn.lock` that need to be committed.

**Comment by Bdthomson** [comment]
image

I noticed while playing with this that dismissing alerts doesn't seem to work unless you click it 3 times.

**Comment by andrewkfiedler** [comment]
Yeah, the alerts issue is one I noticed as well.  Not sure if we should lie and not refetch alerts on dismiss for now, or we should look into why sending a dismiss isn't always working.  

Basically we send an request to delete the alert, refetch, and the alert is still there.

**Comment by Bdthomson** [comment]
Soft commit?

**Comment by andrewkfiedler** [comment]
Could be, I did notice they eventually disappear.

**Comment by andrewkfiedler** [comment]
build now

**Comment by andrewkfiedler** [comment]
build now

File: ui/pom.xml [hunk 3]
```diff
@@ -371,6 +371,10 @@
                                     <classifier>security-certificate-adminmodule</classifier>
                                 </artifact>
                                 <artifact>
+                                    <file>packages/beta-admin-ui/target/beta-admin-ui.jar</file>
+                                    <type>jar</type>
+                                    <classifier>beta-admin-ui</classifier>
+                                </artifact>
```
Comment (andrewkfiedler) [inline ui/pom.xml]:
```suggestion
                                </artifact>
                                <artifact>
```

**Comment by andrewkfiedler** [comment]
build now

**Comment by blen-desta** [comment]
Hero:
- Modified a configuration in every app and verified that the change was reflected in both UIs
- Verified basic operations in Intrigue work
- Tested with a reverse proxy 
- Verified that installing and uninstalling features

**Comment by andrewkfiedler** [comment]
build now

**Comment by andrewkfiedler** [comment]
build now

**Comment by andrewkfiedler** [comment]
cve build

**Comment by andrewkfiedler** [comment]
build now

**Comment by blen-desta** [comment]
✅ Re-hero
- Verified search bar works as intended
- Able to save configurations
- Logo takes to the landing page

**Pull request merged**